docker/docker_image_src.go

```diff
@@ -400,26 +400,23 @@ func (s *dockerImageSource) GetSignaturesWithFormat(ctx context.Context, instanc
 	switch {
 	case s.c.supportsSignatures:
 		sigs, err := s.getSignaturesFromAPIExtension(ctx, instanceDigest)
+		res = append(res, sigs...)
 		if err != nil {
-			return nil, err
+			return res, err
```

Comment:
and we may not even want to fail-fast here, in case `getSignaturesFromSigstoreAttachments` has any luck finding signatures a few lines down.  If we go that route, we would want to collect and aggregate errors from any troubled strategies.